.github/pull_request_template.md

```diff
@@ -0,0 +1,15 @@
+## What is this PR doing?
+
+
+## How can these changes be manually tested?
+
+
+## Does this PR resolve or contribute to any issues?
+
+
+## Checklist
+- [ ] I have manually tested these changes
+- [ ] This PR is ready for review
```

Comment:
this point feels redundant imo as being a "draft" implies it's not ready for review and once you _un-draft_ a PR it says "marked ready for review" by default.

Reply:
Ah yeah good point

Reply:
Done

(Btw, another suggestion I have is that the author shouldn't click 'resolve conversation' - I think it's up to the person who raised the point to mark it as resolved.)

Reply:
Done